docs/metachromium-overview.md

```diff
@@ -3,5 +3,54 @@ id: metachromium-overview
 title: Metachromium Overview
 ---
 
-Metachromium
+[Metachromium](https://metachromium.com/) is a spatial app browser that runs side-by side with OpenVR. It lets you open any WebXR page as a spatial overlay app that runs on top of any other VR game.
 
+## Features
+
+- Run WebXR context on top of an OpenVR application
+
+- Run multiple WebXR applications inside a single Metachromium context
+  
+- Add multiplayer to singleplayer apps using metaRTC
+
+- Volumetric streaming capabilities
+
+## Using Metachromium
+
+### Prerequisites
+
+- XR Device (for best experience)
+
+- At least 1GB of storage
+
+### Running
+
+1. Install Metachromium through [Steam](https://store.steampowered.com/app/685110/Metachromium/). If you need an early-access key, get one [here](https://metachromium.com/).
+
+2. Run your SteamVR application.
+
+3. Run Metachromium and choose an app you would like to overlay.
+
+4. Enjoy!
+
+## How It Works
+
+Metachromium is based on the open source Chromium browser and is based on web standards like WebXR.
+
+### Rendering Content
+
+For different WebXR contexts to be overlayed inside an OpenVR application, special math must be done so the outlines of these different applications blend into the world correctly in terms of its alpha (opacity). Metachromium does the blending such that the renderings appear as though they are in the correct space when viewed through the headset. Functionally, there are a bunch of 3D geometry calculations that are done under the hood to make an immersive experience possible.
+
+### Using XRPackages
+
+One of the caveats with overlaying WebXR objects is that all WebXR content appears as though it's overlayed on top of the native app, instead of appearing as though it were native to the application. This is one of the main reasons why using XRPackages is preferred, as a `.wbn` generated by XRPackage contains the correct depth rendering information needed for a more immersive rendering.
+
+### Further Reading
+
+To learn more about Metachromium and how to develop your own content, follow our [create your first aug](creating-an-aug.md) tutorial. Other useful links to learning more about the Webaverse, XRPackage, and XR development can be found below.
```

Comment:
For the sake of styling consistency, let's wrap all page links that use the actual page title in backticks (`example`) and capitalize case the link text to reflect the title.

Reply:
Hey @Zetaphor just to confirm, did you mean something like
```markdown
follow our [`Create Your First Aug`](creating-an-aug.md) tutorial.
```
So it shows up like [`Create Your First Aug`](creating-an-aug.md)?

Reply:
Correct, see [this section on the main page](https://docs.webaverse.com/docs/index.html#import-existing-creations) for an example in production